app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,61 @@
+<br><br>
+<div class='center'>
 <h2>Log in</h2>
+  <style>
```

Comment:
Putting `<style>` blocks into a `.html.erb` file is a handy way to figure out what the CSS should be.  But once you've settled on it, it would be good to move the CSS into a stylesheet `.css` file.  

There are two approaches we could take:  

1) You could move this CSS to be in `app/assets/stylesheets/application.css` instead.  This means that the CSS rules would be in effect on every page in the app.  This might be what you want, or it might not.  

2) If you only want these rules to be in effect on this page, you can put it into a *new* CSS file in `app/assets/stylesheets` and give it a name like `log_in.css`.  Then, on this page, you would need to include that CSS using a `stylesheet_link_tag`.  Here is some documentation about how to do that: https://apidock.com/rails/ActionView/Helpers/AssetTagHelper/stylesheet_link_tag